azurerm/internal/services/containers/kubernetes_cluster_node_pool_resource.go

```diff
@@ -39,6 +40,8 @@ func resourceArmKubernetesClusterNodePool() *schema.Resource {
 			Delete: schema.DefaultTimeout(60 * time.Minute),
 		},
 
+		CustomizeDiff: customizeDiff,
```

Comment:
can we remove the customizeDiff here? unfortunately this is broken when a field is interpolated (most of the time)